new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.js

```diff
@@ -1,7 +1,7 @@
 import { makeStyles } from '@material-ui/core/styles'
 import classnames from 'classnames'
 import { useFormikContext, Form, Formik, Field as FormikField } from 'formik'
-import _ from 'lodash'
+import _ from 'lodash/fp'
```

Comment:
Hmm, just noticing that this was using lodash, we don't even have lodash on the front-end build scripts. Replace this with ramda.